code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -449,29 +461,23 @@
 		if(INTENT_HELP, INTENT_GRAB, INTENT_DISARM) //Disperse
 			for(var/i in 1 to illusion_count)
 				illusions += new /mob/illusion/xeno(owner.loc, owner, null, illusion_life_time)
-	addtimer(CALLBACK(src, .proc/clean_illusions), illusion_life_time)
```

Comment:
I don't see the point of moving that

Reply:
Was for clarity on how cooldown was handled by caller, but moved back due to other simplifications